File: lc1512.js
var numIdenticalPairs = (nums)=>{
    let n = nums.length;
    let cnt = 0;
    for(let i=0;i<n-1;i++){
        for(let j=i+1;j<n;j++){
            cnt = (nums[i]===nums[j])?cnt+1:cnt+0;
        }
    }
    return cnt;
};

File: py/lc1512.py
def numIdenticalPairs(nums):
    n = len(nums)
    cnt = 0
    for i in range(n-1):
        for j in range(i+1, n):
            if nums[i]== nums[j]:
                cnt = cnt+1
    return cnt

nums = [1,2,3,1,1,3]
print(numIdenticalPairs(nums))
